src/js/vic-v2/components/IntroductionPage.jsx

```diff
@@ -1,16 +1,34 @@
 import React from 'react';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
+import Scroll from 'react-scroll';
 
 import { focusElement } from '../../common/utils/helpers';
 import FormTitle from '../../common/schemaform/components/FormTitle';
 import SaveInProgressIntro, { introActions, introSelector } from '../../common/schemaform/save-in-progress/SaveInProgressIntro';
+import { hasSavedForm } from '../helpers';
+
+const { animateScroll: scroll } = Scroll;
 
 class IntroductionPage extends React.Component {
   componentDidMount() {
     focusElement('.va-nav-breadcrumbs-list');
   }
 
+  componentDidUpdate(prevProps) {
+    const { savedForms } = this.props.saveInProgress.user.profile;
+    const { formId } = this.props.saveInProgress;
+    const notPreviouslyLoggedIn = !prevProps.saveInProgress.user.login.currentlyLoggedIn;
+    if (notPreviouslyLoggedIn && hasSavedForm(savedForms, formId)) {
+      focusElement('[id="2-continueButton"]', { preventScroll: true });
+      scroll.scrollToTop(window.VetsGov.scroll || {
```

Comment:
what is `window.VetsGov`?

Reply:
It's a hackish way of letting us disable smooth scrolling in e2e tests. We set that global object in e2e tests and leave it empty in other environments. We just started disabling `window.scrollTo` entirely in e2e tests, so it may go away in the future.